packages/lit-dev-content/site/tutorials/content/word-viewer/00.md

```diff
@@ -30,9 +30,10 @@ concepts you'll use when building your own Lit components:
 
 # Previous knowledge
 
-This tutorial assumes a basic familiarity with JavaScript or TypeScript, as well
-as some basics of Lit covered in the [Intro to Lit
-Tutorial](/tutorials/intro-to-lit).
+This tutorial assumes a basic familiarity with&nbsp;
+<ts-js><span slot="js">JavaScript</span><span slot="ts">TypeScript</span></ts-js>,
+as well as some basics of Lit covered in the
+[Intro to Lit Tutorial](/tutorials/intro-to-lit).
```

Comment:
I think this is overkill; If someone clicks through and sees "... assumes a basic familiarity with TypeScript," they may just stop there and not realize there's a switch.